### DISCUSSION TITLE
이거풀게요 12월 둘째주

### DISCUSSION THREAD

**Discussion by abcdsds** [brewagebear/ISOLVEDTHEPROBLEM#2] [금주의 문제]
https://programmers.co.kr/learn/courses/30/lessons/42576

https://programmers.co.kr/learn/courses/30/lessons/42840

**Comment by Tianea2160** [comment]
https://programmers.co.kr/learn/courses/30/lessons/42862
https://programmers.co.kr/learn/courses/30/lessons/82612

**Comment by brewagebear** [comment]
1. [로또의 최고 순위와 최저 순위](https://programmers.co.kr/learn/courses/30/lessons/77484)
2. [신규 아이디 추천](https://programmers.co.kr/learn/courses/30/lessons/72410)
3. [숫자 문자열과 영단어](https://programmers.co.kr/learn/courses/30/lessons/81301)
4. [키패드 누르기](https://programmers.co.kr/learn/courses/30/lessons/67256)

**Reply by brewagebear** [reply]
[풀이 완료](https://github.com/dailyworker/ISOLVEDTHEPROBLEM/tree/main/dailywokrer)

**Comment by pch8388** [comment]
https://programmers.co.kr/learn/courses/30/lessons/86051
https://programmers.co.kr/learn/courses/30/lessons/64061

**Comment by Hwanse** [comment]
- [프로그래머스1레벨_2020 카카오 인턴십_키패드 누르기](https://programmers.co.kr/learn/courses/30/lessons/67256)
- [프로그래머스1레벨_2021 카카오 채용연계형 인턴십_숫자 문자열과 영단어](https://programmers.co.kr/learn/courses/30/lessons/81301)

**Reply by Hwanse** [reply]
문제 풀이 PR 올렸습니다
https://github.com/dailyworker/ISOLVEDTHEPROBLEM/pull/4

**Reply by brewagebear** [reply]
머지 완료했습니다.

**Comment by csj4032** [comment]
프로그래머스1레벨_2020 카카오 인턴십_키패드 누르기
프로그래머스1레벨_2021 카카오 채용연계형 인턴십_숫자 문자열과 영단어
https://programmers.co.kr/learn/courses/30/lessons/86051
https://programmers.co.kr/learn/courses/30/lessons/64061
https://programmers.co.kr/learn/courses/30/lessons/42862
https://programmers.co.kr/learn/courses/30/lessons/82612
https://programmers.co.kr/learn/courses/30/lessons/42576
https://programmers.co.kr/learn/courses/30/lessons/42840

**Comment by abcdsds** [comment]
> https://programmers.co.kr/learn/courses/30/lessons/42576
> 
> https://programmers.co.kr/learn/courses/30/lessons/42840

[풀이완료](https://github.com/abcdsds/programmers/tree/main/src/level1)
